node/src/components/consensus/utils.rs

```diff
@@ -139,6 +139,8 @@ pub(crate) fn check_sufficient_finality_signatures(
     )
 }
 
+// TODO - remove the `allow` below.
+#[allow(unused)]
 pub(crate) fn get_minimal_set_of_signatures(
     trusted_validator_weights: &BTreeMap<PublicKey, U512>,
     finality_threshold_fraction: Ratio<u64>,
```

Comment:
I know it hasn't been introduced in this PR, but maybe we can take this opportunity to fix what has been bothering me for a while now, since some other things are also being renamed - I believe the name of this variable is based on a mistaken expansion of "FT" into "finality threshold", when it _should_ mean "fault tolerance" in this context (@afck please correct me if I'm wrong!). So, should we rename the variable to "fault_tolerance_fraction" (and also the one in the fetcher, probably)?

Reply:
I agree! :+1: 
(I think at the time we _did_ mean to say "the (fault tolerance) threshold beyond which we declare a block to be final" — but just "fault tolerance" makes much more sense!)

Reply:
I renamed the instances of this variable which are affected by other mechanical changes in this PR (mainly moving this file from consensus to linear_chain).  We can probably rename the remainder if others agree, but I don't want to increase the size of this PR too much further if possible.